post_process_transforms/UTK_ir_etds_post_process.xsl

```diff
@@ -23,8 +23,17 @@
   </xsl:template>
 
   <!-- if any of the following elements are empty, drop them from the transform. this list could grow. -->
-  <!-- if an extra thesis advisor or committee member is added to the form -->
-  <xsl:template match="mods:name[@type='personal'][mods:displayForm='']"/>
+  <!--
+    if a thesis advisor or committee member is added via the form without a namePart[@type='given' AND 'family'] AND
+    has a role/roleTerm='Thesis advisor' or 'Committee member'.
+    MODS added via the form automatically gets an empty mods:displayForm.
+  -->
+  <xsl:template match="mods:name[mods:displayForm='']
```

Comment:
We need to have a better conditional here.  If something has a blank displayForm but no nameParts, there will not be a match.

Reply:
@markpbaggett @cdeaneGit please take a look at the latest changes - I think I've updated this conditional appropriately.

Reply:
I started looking at this yesterday.  This was after 4, so I pulled code after you changes were in place. Still testing.

Reply:
@cdeaneGit and @markpbaggett please review the updated testing instructions in the pull request. Let me know if you have any questions. 

Thanks!